doc/releasenotes/15_0_0_release_notes.md

```diff
@@ -1,47 +1,47 @@
-- **[Command-Line Syntax Deprecations](#a-idcommand-line-syntax-deprecationscommand-line-syntax-deprecations)**
-  - [VTTablet Startup Flag Deletions](#a-idvttablet-startup-flag-deletionsvttablet-startup-flag-deletions)
-  - [VTTablet Startup Flag Deprecations](#a-idvttablet-startup-flag-deprecationsvttablet-startup-flag-deprecations)
-  - [VTBackup Flag Deprecations](#a-idvtbackup-flag-deprecationsvtbackup-flag-deprecations)
-- **[VTGate](#a-idvtgatevtgate)**
-  - [vtgate --mysql-server-pool-conn-read-buffers](#a-idvtgate--mysql-server-pool-conn-read-buffersvtgate---mysql-server-pool-conn-read-buffers)
-- **[VDiff2](#a-idvdiff2vdiff-v2)**
-  - [Resume Workflow](#a-idresume-workflowresume-workflow)
-  - [vtctl GetSchema --table-schema-only](#a-idvtctl-getschema--table-schema-onlyvtctl-getschema---table-schema-only)
-  - [Support for Additional Compressors and Decompressors During Backup & Restore](#a-idsupport-for-additional-compressors-and-decompressors-during-backup--restoresupport-for-additional-compressors-and-decompressors-during-backup--restore)
-  - [Independent OLAP and OLTP Transactional Timeouts](#a-idindependant-olap-and-oltp-transactional-timeoutsindependent-olap-and-oltp-transactional-timeouts)
-  - [Support for Specifying Group Information in Calls to VTGate](#a-idsupport-for-specifying-group-information-in-calls-to-vtgatesupport-for-specifying-group-information-in-calls-to-vtgate)
-- **[Online DDL Changes](#a-idonline-ddl-changesonline-ddl-changes)**
-  - [Concurrent Vitess Migrations](#a-idconcurrent-vitess-migrationsconcurrent-vitess-migrations)
-  - [VTCtl Command Changes](#a-idvtctl-command-changesvtctl-command-changes)
-  - [New Syntax](#a-idnew-syntaxnew-syntax)
-- **[Tablet Throttler](#a-idtablet-throttlertablet-throttler)**
-  - [API Changes](#a-idapi-changesapi-changes)
-- **[Mysql Compatibility](#a-idmysql-compatibilitymysql-compatibility)**
-  - [System Settings](#a-idsystem-settingssystem-settings)
-  - [Lookup Vindexes](#a-idlookup-vindexeslookup-vindexes)
-- **[Durability Policy](#a-iddurability-policydurability-policy)**
-  - [Cross Cell](#a-idcross-cellcross-cell)
-- **[New EXPLAIN Format](#a-idnew-explain-formatnew-explain-format)**
-  - [FORMAT=vtexplain](#a-idformatvtexplainformatvtexplain)
-- **[VTOrc](#a-idvtorcvtorc)**
-  - [Old UI Removal and Replacement](#a-idold-ui-removal-and-replacementold-ui-removal-and-replacement)
-  - [Configuration Refactor and New Flags](#a-idconfiguratoin-refactor-and-new-flagsconfiguration-refactor-and-new-flags)
-  - [Example Upgrade](#a-idexample-upgradeexample-upgrade)
-  - [Default Configuration Files](#a-iddefault-configuration-filesdefault-configuration-files)
-- **[Flags Restructure](#a-idflags-restructureflags-restructure)**
-  - [Flags Diff](#a-idflag-diffflags-diffd)
+- **[Breaking Changes](#breaking-changes)**
+  - [Flags](#flags)
+  - [VTTablet Flag Deletions](#vttablet-flag-deletions)
+  - [Vindex Interface](#vindex-interface)
+- **[Deprecations](#deprecations)**
```

Comment:
If it's not too late, we could change this to deprecations and removals and also note the removal of all legacy sharding related code: https://github.com/vitessio/vitess/pull/10278